openassessment/xblock/test/test_grade.py

```diff
@@ -327,7 +327,7 @@ def test_submit_feedback_invalid_options(self, xblock):
             'feedback_text': u'I disliked my assessment',
             'feedback_options': u'should be a list!',
         })
-        resp = self.request(xblock, 'submit_feedback', payload, response_format='json')
+        resp = self.request(xblock, 'submit_feedback', payload, response_format='json', use_runtime=False)
```

Comment:
The runtime does a `save()` on the XBlock after the request is complete which messes up the transaction management. See http://stackoverflow.com/a/23326971.